.rspec

```diff
@@ -1,2 +0,0 @@
---color
```

Comment:
This has been updated to use `--color` and `--format documentation`.  If you like dots, you can use `--format progress`.  If you really really like dots, I can change it back.  But you'll owe me bigtime.